client/packages/android/app/src/main/java/org/openmsupply/client/CertWebViewClient.java

```diff
@@ -199,10 +199,14 @@ public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest request
         // will not only reload, but will open the URL in a browser tab
         // for local URLs we don't want this to happen!
         Uri url = request.getUrl();
-        if (url.toString().startsWith(nativeApi.getServerUrl())) {
-            return false;
+        try {
+            if (url.toString().startsWith(nativeApi.getServerUrl())) {
+                return false;
+            }
+        }
+        catch(Exception e){
+            Log.e(TAG,e.getMessage());
         }
-
         return bridge.launchIntent(url);
```

Comment:
> CertWebViewClient.shouldOverrideUrlLoading was raising an Exception about String.length() of a null object

Hmm i wonder if the logic here means that when running android app without network, and exception will be called and we will open URL in new browser window ?